ui/css/itcss/tools/utilities.scss

```diff
@@ -1,37 +0,0 @@
-/*
-  Utility Classes
- */
-
-/* lib */
-.flex-column {
-  display: flex;
-  flex-direction: column;
-}
```

Comment:
`flex-column` class is not used

![Screenshot 2025-02-07 at 2 57 54 PM](https://github.com/user-attachments/assets/64fa0c77-a86e-4d35-a6cd-de87bd7abf86)